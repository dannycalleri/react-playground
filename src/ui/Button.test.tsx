import { render } from "@testing-library/react";
import { Button } from ".";

describe("Button", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<Button type="submit">Send</Button>);
    expect(asFragment()).toMatchSnapshot();
  });

  it("should render correctly with different type", () => {
    const { asFragment } = render(<Button type="button">Send</Button>);
    expect(asFragment()).toMatchSnapshot();
  });
});
