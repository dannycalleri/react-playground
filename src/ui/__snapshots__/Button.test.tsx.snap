// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<DocumentFragment>
  <button
    type="submit"
  >
    Send
  </button>
</DocumentFragment>
`;

exports[`Button should render correctly with different type 1`] = `
<DocumentFragment>
  <button
    type="button"
  >
    Send
  </button>
</DocumentFragment>
`;
